.env.example

```diff
@@ -6,11 +6,12 @@ GIN_MODE=debug
 # The port on which the api will listen on (default: 3000)
 PORT=3000
 
-
 # Database Related Config
 
 DB_HOST=localhost
```

Comment:
for consistency we should go with `POSTGRES_HOST` and `POSTGRES_PORT`, they should not conflict with the postgres docker image (at least they are not mentioned in the [docs](https://hub.docker.com/_/postgres/))